api/pom.xml

```diff
@@ -30,7 +30,9 @@
 
   <name>Microsoft Entra ID SSO API</name>
   <description>This extension allows users to authenticate to the wiki using Entra ID, formerly known as Azure Active Directory.</description>
-
+  <properties>
+    <xwiki.extension.features>com.xwiki.identity-oauth:identity-oauth-api</xwiki.extension.features>
```

Comment:
A comment would be nice, to understand why this is needed (to mention that this module needs to be uninstalled now that is no longer used, and his functionalities are actually provided by oidc)